byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Block.hs

```diff
@@ -70,10 +71,16 @@ data BlockBody
   -- ^ Protocol version
   } deriving (Generic, Show)
 
-instance HasTypeReps BlockBody
-
 makeLenses ''BlockBody
 
+instance HasTypeReps BlockBody where
+  typeReps b = typeOf b
+               <| typeOf (b ^. bUpdProp :: Maybe UProp)
```

Comment:
Here update proposal has a set of `STag` inside

```haskell
data UProp = UProp
  {
  , _upSTags :: Set STag
```

so this will return a constant type `Maybe UProp`. We need the list with the `STag` types as well

Reply:
I've added a "cost" to STag in `bBodySize`